lib/active_shipping/carriers/stamps.rb

```diff
@@ -167,11 +167,16 @@ def find_tracking_info(shipment_id, options = {})
       commit(:TrackShipment, request)
     end
 
-    def cancel_shipment(tracking, options = {})
-      request = build_cancel_indicium_request(tracking, options)
+    def cancel_shipment(shipment_id, options = {})
```

Comment:
I noticed in the Food52 repo it does grab `:tracking_number` when calling `void_labels`. Is `shipment_id` a naming convention we'll move to instead of tracking numbers?

Reply:
Most Stamps requests that deal with "shipments" are able to use either a StampsTxnId or TrackingNumber. It seems like most of the public methods accept a generic ID (called shipment_id) and based on the options passed along side it determine if it should be used as a StampsTxnId or a TrackingNumber (https://github.com/food52/active_shipping/blob/2f17db6c09a43970569f93a7a3e09e8da6119d36/lib/active_shipping/carriers/stamps.rb#L455) so I was just moving toward that pattern.